modelmapper/signature.py

```diff
@@ -71,5 +71,7 @@ def get_byte_str_of_row(row, ignore_fields=[]):
     return row_bytes
 
 
-def get_hash_of_bytes(item):
+def get_hash_of_bytes(item, bit32=False):
```

Comment:
Can we place change this to be `bits=64` as default and then when you override it with 32, it runs the `hash`?
If you make a mapping of bit to the function like:

```
{
32: mm3.hash,
64: mm3.hash64,
128: mm3.hash128
}
```

Then it can grab the right function to run.  Also can you please tell me agin why we have `x64arch=False` for 64?

Reply:
x64arch is to optimize between x86 and x64 operating systems and it was chosen to optimize for x86 in our system.